specification/src/main/asciidoc/webprofile/Introduction.adoc

```diff
@@ -68,7 +68,11 @@ core (“à la carte” installation).
 
 === Determining Applicable Requirements
 
-*
+NOTE: Profile definitions can be quite terse, amounting to little more than a list of required technologies and a (possibly empty) set of additional requirements, beyond those entailed by all the referenced specifications.
+Being the first profile of the Java(TM) EE 6 Platform to be defined, we expect the Web Profile specification to be used as a model for future profiles.
+It will also be seen as a starting point for understanding how the requirements defined in the Jakarta EE Platform specification apply to a profile that subsets the platform itself, a significant innovation in this version of the platform.
+(The case of a profile that is a superset of the platform is much easier to picture.)
+To help with this process, this section attempts to shed light on how one should go from the definition of the Web Profile to figuring out Jakarta EE 8 Web Profile, the exact set of requirements that apply to it, and consequently to any product that implements it.
```

Comment:
Is this all new text?  Where did it come from?

It could stand some rewrites to be more relevant to the current version.

Reply:
Yes, it came from the Final Release of the Jakarta EE 8 Web Profile document.  I was reviewing page by page of the Web Profile spec and came across this missing text.  In the adoc file, this missing section had a single asterisk in its place.  At this point, I think it's best to have the Jakarta EE Specs match the content of the Java EE Specs, except for the Java->Jakarta and the acronym usage.  Any additional type of re-wording should not be part of Jakarta EE 8.